core/common/src/main/java/alluxio/exception/ExceptionMessage.java

```diff
@@ -36,7 +36,7 @@
   CANNOT_REQUEST_SPACE("Not enough space left on worker {0} to store blockId {1,number,#}."),
   NO_LOCAL_WORKER("Local address {0} requested but there is no local worker"),
   NO_WORKER_AVAILABLE_ON_ADDRESS("No Alluxio worker available for address {0}"),
-  NO_WORKER_AVAILABLE("No available Alluxio worker found"),
+  NO_WORKER_AVAILABLE("There is no worker with enough space for a new block of size ${blocksize}"),
```

Comment:
This message is used in a couple different situations, some of which aren't related to blocks. Could we leave this message as-is, and use a new message called `NO_SPACE_FOR_BLOCK_ON_WORKER` in `getOutStream`?